client/src/game/ui/action.ts

```diff
@@ -508,7 +508,8 @@ actionFunctions.set('strike', (data: ActionStrike) => {
   const strikeX = globals.elements.strikeXs[i];
 
   // Update the stats
-  globals.cluesSpentPlusStrikes += 1;
+  // In clue starved variants, each strike only "costs" half a clue
+  globals.cluesSpentPlusStrikes += globals.variant.name.startsWith('Clue Starved') ? 0.5 : 1;
```

Comment:
variantRules.isClueStaved(variant)
Let's not add more string based comparisons now that there is a better way

Reply:
is that implemented already? i would expect the rest of the code to be refactored to use this

Reply:
variants.ts line 7

Reply:
I'll log an issue to replace this kind of thing everywhere